cypress/integration/shared.js

```diff
@@ -17,8 +17,18 @@ export const clusterNameLinkSelector = '[data-label="Name"] > a'; // on '/cluste
 // const singleClusterCellSelector = (column) => `tbody > tr > [data-label="${column}"]`;
 export const clusterTableCellSelector = (row, column) =>
   `tbody > tr:nth-child(${row}) > [data-label="${column}"]`;
+export const hostDetailSelector = (row, label) =>
+  // NOTE: The first row is number 2! Shift your indexes...
+  `table > tbody:nth-child(${row}) > tr:nth-child(1) > [data-label="${label}"]`;
 
 export const PULL_SECRET = Cypress.env('PULL_SECRET');
+export const SSH_PUB_KEY = Cypress.env('SSH_PUB_KEY');
+export const CLUSTER_NAME = Cypress.env('CLUSTER_NAME');
+export const DNS_DOMAIN_NAME = Cypress.env('DNS_DOMAIN_NAME');
+export const API_VIP = Cypress.env('API_VIP');
+export const INGRESS_VIP = Cypress.env('INGRESS_VIP');
+export const NUM_MASTERS = parseInt(Cypress.env('NUM_MASTERS'));
```

Comment:
Please ad reasonable defaults to `/cypress.json`, otherwise this is `NaN`

Reply:
Related: https://github.com/openshift-metal3/facet/pull/269

Reply:
Done